app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -1297,6 +1297,126 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_interactions_noItemSelected_selectionTextIsDisplayed() {
+    launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use {
+      startPlayingExploration()
+      playThroughPrototypeState1()
+      playThroughPrototypeState2()
+      playThroughPrototypeState3()
+      playThroughPrototypeState4()
+
+      // Verify that the attributes required for correct accessibility support are present.
+      verifyViewTypeIsPresent(SELECTION_INTERACTION)
+      onView(withId(R.id.selection_interaction_textview)).check(matches(isDisplayed()))
+    }
+  }
+  @Test
+  fun testStateFragment_interactions_land_noItemSelected_selectionTextIsDisplayed() {
+    launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use {
+      rotateToLandscape()
+      startPlayingExploration()
+      playThroughPrototypeState1()
+      playThroughPrototypeState2()
+      playThroughPrototypeState3()
+      playThroughPrototypeState4()
+
+      // Verify that the attributes required for correct accessibility support are present.
+      verifyViewTypeIsPresent(SELECTION_INTERACTION)
+      onView(withId(R.id.selection_interaction_textview)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testStateFragment_interactions_singleItemSelected_selectionTextIsNotDisplayed() {
+    launchForExploration(TEST_EXPLORATION_ID_2, shouldSavePartialProgress = false).use {
+      startPlayingExploration()
+      playThroughPrototypeState1()
+      playThroughPrototypeState2()
+      playThroughPrototypeState3()
+      playThroughPrototypeState4()
+
+      selectItemSelectionCheckbox(optionPosition = 0, expectedOptionText = "Red")
+      // Verify that the attributes required for correct accessibility support are present.
+      verifyViewTypeIsPresent(SELECTION_INTERACTION)
+      onView(withId(R.id.selection_interaction_textview)).check(matches(not(isDisplayed())))
+    }
+  }
+
+  @Test
+  fun testStateFragment_interactions_land_singleItemSelected_selectionTextIsNotDisplayed() {
```

Comment:
Can you add tests to demonstrate that the selection text retains its showing/not showing behavior after performing a rotation & activity recreation?

Reply:
@BenHenning Yes. I will add the test case.

Reply:
@BenHenning I have remove the landscape test cases as these doesn't make sense because currently when you select some answers and rotate the screen all values gets lost. So right now we can't add those test cases related to orientation change or activity recreation.
Follow are the removed test cases for landscape orientation:
`testStateFragment_interactions_land_noItemSelected_selectionTextIsDisplayed`
`testStateFragment_interactions_land_singleItemSelected_selectionTextIsNotDisplayed`
`testStateFragment_interactions_land_multipleItemSelected_selectionTextIsNotDisplayed`
`testStateFragment_interactions_changeConfiguration_selectUnselectCheckbox_selectionTextIsDisplayedCorrectly`